.repo-metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "tasks",
+  "name_pretty": "Cloud Tasks",
+  "product_documentation": "https://cloud.google.com/tasks/docs/",
+  "client_documentation": "https://cloud.google.com/nodejs/docs/reference/tasks/latest/",
+  "issue_tracker": "",
```

Comment:
There totally is one!  This page is your friend:
https://cloud.google.com/support/docs/issue-trackers